cpp/dolfinx/io/vtk_utils.cpp

```diff
@@ -20,18 +20,21 @@ using namespace dolfinx;
 
 //-----------------------------------------------------------------------------
 std::pair<std::vector<std::int64_t>, std::array<std::size_t, 2>>
-io::extract_vtk_connectivity(const graph::AdjacencyList<std::int32_t>& dofmap_x,
-                             mesh::CellType cell_type)
+io::extract_vtk_connectivity(
+    std::experimental::mdspan<const std::int32_t,
+                              std::experimental::dextents<std::size_t, 2>>
+        dofmap_x,
+    mesh::CellType cell_type)
 {
   // Get DOLFINx to VTK permutation
   // FIXME: Use better way to get number of nodes
   // const std::size_t num_nodes = geometry.cmap().dim();
-  const std::size_t num_nodes = dofmap_x.links(0).size();
+  const std::size_t num_nodes = dofmap_x.extent(1);
```

Comment:
I guess you now can remove lines 30-31, as this would be a correct way of getting the number of nodes.